docs/podman-login.1.md

```diff
@@ -33,6 +33,10 @@ Username for registry
 
 Path of the authentication file. Default is ${XDG_\RUNTIME\_DIR}/containers/auth.json
 
+**--get-login**
```

Comment:
Can we move this below? Currently, it splits the `Note: [...]` that relates to `--authfile`.

Reply:
fixed

Reply:
Thanks! The latest commit isn't signed. Could you re-submit (e.g., `git commit --amend -s` and push)? Then we can merge :)

Reply:
also fixed